src/main/kotlin/org/jetbrains/research/ml/ast/transformations/PyUtils.kt

```diff
@@ -0,0 +1,19 @@
+package org.jetbrains.research.ml.ast.transformations
+
+import com.jetbrains.python.psi.LanguageLevel
+import com.jetbrains.python.psi.PyElementGenerator
+import com.jetbrains.python.psi.PyIfPart
+import com.jetbrains.python.psi.PyIfStatement
+
+object PyUtils {
+    fun createPyIfElsePart(ifElsePart: PyIfPart): PyIfPart {
+        assert(ifElsePart.isElif)
```

Comment:
Let's use assert only for debugging and not to push it to master. Use `require` if you need to